app/src/main/java/com/github/lookupgroup27/lookup/ui/profile/Profile.kt

```diff
@@ -1,14 +1,102 @@
+import com.github.lookupgroup27.lookup.ui.navigation.Screen
 
+/**
+ * A Composable function that renders the Profile screen, which includes:
+ * - A profile icon at the top center of the screen.
+ * - Two buttons: "Personal Info" and "Your Collection," both centered below the profile icon.
+ * - A bottom navigation menu to navigate between different sections (e.g., Map, Menu).
+ *
+ * The buttons trigger navigation actions when clicked, allowing users to navigate to the Profile or
+ * Collection screens.
+ *
+ * @param navigationActions An instance of [NavigationActions] that defines the navigation behavior.
+ *   It handles navigating between the Profile, Collection, and other top-level sections of the app
+ *   through the bottom navigation menu.
+ */
 @Composable
 fun ProfileScreen(navigationActions: NavigationActions) {
-  SampleScreen(
-      screenText = "Profile Screen",
-      navigationActions = navigationActions,
-      screenTag = "profile_screen",
-      backButtonTag = "go_back_button_profile")
+  // Column to stack the profile section and bottom navigation
+  Column(
+      modifier =
+          Modifier.fillMaxSize().background(Color.DarkGray), // Background for the entire screen
+      verticalArrangement =
+          Arrangement.SpaceBetween // Ensure items are spaced between top and bottom
+      ) {
+        // Center the Profile Icon and Buttons in the middle of the screen
+        Column(
+            modifier = Modifier.fillMaxWidth().padding(top = 300.dp), // Padding from the top
+            horizontalAlignment = Alignment.CenterHorizontally // Center content inside the column
+            ) {
+              // Profile Icon
+              Icon(
+                  imageVector = Icons.Default.AccountCircle,
+                  contentDescription = "Profile Icon",
+                  tint = Color.White,
+                  modifier = Modifier.size(150.dp))
+
+              // Spacer between profile icon and first button
+              Spacer(modifier = Modifier.height(60.dp))
+
+              // Personal Info Button
+              Button(
+                  onClick = { navigationActions.navigateTo(Screen.PROFILE) },
+                  colors =
+                      ButtonDefaults.buttonColors(
+                          containerColor = MaterialTheme.colorScheme.primary,
+                          contentColor = Color.White),
+                  modifier = Modifier.padding(vertical = 8.dp)) {
+                    Text(text = "Personal Info", fontSize = 18.sp)
+                  }
+
+              // Collection Button
+              Button(
+                  onClick = { navigationActions.navigateTo(Screen.COLLECTION) },
+                  colors =
+                      ButtonDefaults.buttonColors(
+                          containerColor = MaterialTheme.colorScheme.primary, // Solid button color
+                          contentColor = Color.White),
+                  modifier = Modifier.padding(vertical = 8.dp)) {
```

Comment:
It looks like both of these buttons for the "Profile" and "Collection" have very similar implementations. You can refactor this code into a single composable function to avoid repetition and make the code easier to maintain in the future. Here’s a suggestion for refactoring:

```kotlin
@Composable
fun ProfileButton(text: String, onClick: () -> Unit) {
    Button(
        onClick = onClick,
        colors = ButtonDefaults.buttonColors(
            containerColor = MaterialTheme.colorScheme.primary,
            contentColor = Color.White
        ),
        modifier = Modifier.padding(vertical = 8.dp)
    ) {
        Text(text = text, fontSize = 18.sp)
    }
} 

// And change the Buttons() with : 

ProfileButton(text = "Personal Info", onClick = { navigationActions.navigateTo(Screen.PROFILE) })
ProfileButton(text = "Your Collection", onClick = { navigationActions.navigateTo(Screen.COLLECTION) })